accounting/serializers.py

```diff
@@ -3,6 +3,11 @@
 
 from .models import Project
 
+from history.models import Standup, Blocker
+
+from datetime import datetime, timedelta
```

Comment:
re-order the imports.

- group by package in this order. (python, django, 3rd party package, different modules, local module)
- re-order the files alphabetically